cartridges/int_cloudinary/cartridge/scripts/service/cldMetadata.js

```diff
@@ -64,9 +64,52 @@ function fetchMetadata() {
     return metadataFields;
 }
 
+
+/**
+ * This method uses the service to create metadata fields in cld.
+ *
+ * @param {Array} schema - Schema to be created in Cloudinary
+ * @returns {string} result - The API service response (JSON)
+ */
+function createMetadataSchemeCld(schema) {
+    var cldResponse = { ok: true, message: '', result: {} };
+    var configArgs = {};
+    var result = null;
+
+    configArgs.method = 'POST';
+    configArgs.endPoint = '/metadata_fields';
+
+    try {
+        var service = cldWebService.getService(Prefs.CLD_METADATA_UPLOAD, cldWebService.getServiceConfigs(configArgs));
```

Comment:
The metadata belongs to the uploaded assets, thus, no need for a new service.
Please use the CLD_UPLOAD_SVC

Reply:
The endpoint for CLD_UPLOAD_SVC is "https://api.cloudinary.com/v1_1/" and
the endpoint for CLD_METADATA_UPLOAD is "https://[apikey]:[apisecret]@api.cloudinary.com/v1_1/[cloudname]"
since both services using different endpoints therefore, we have created new service.

Reply:
@asad-rafter This is the same endpoint exactly. You can see how it is used in [fetchMetadataFromCld](https://github.com/cloudinary/cloudinary_sfcc_site_cartridge/blob/3da7d2c15d8e2d175cd2953cb6a8fd138c00272a/cartridges/int_cloudinary/cartridge/scripts/service/cldMetadata.js#L15), as well as in my example.

The way that you defined it forces the user to type again the cloudname, apikey, and apisecret. But this data was already entered in CLD_CLOUDNAME, CLD_REST_SERVICE_CREDENTIALS